p256-4-1/src/views/BasicInfo.vue

```diff
@@ -0,0 +1,218 @@
+        <div class="container__content-question">
+          <div class="container__content-question-title">-性別-</div>
+
+          <div class="control">
+            <input type="radio" name="gender" value="male" id="male"/>
+            <label for="male" class="radio">男性</label>
+
+            <input type="radio" name="gender" value="female" id="female"/>
+            <label for="female" class="radio">女性</label>
+          </div>
+        </div>
+
+        <div class="container__content-question">
+          <div class="container__content-question-title">-生年月日-</div>
+
+          <div class="control container__control">
+            <div class="select" v-on:change="changeDate()">
+              <select id="year" v-model="yearSelected">
+                <option v-for="yearValue in yearValues"
+                        :value="yearValue.year"
+                        :key="yearValue.year">
+                  {{ yearValue.jpYear }}
+                </option>
+              </select>
+            </div>
+            <span class="ymd-text">年</span>
+
+            <div class="select" v-on:change="changeDate()">
+              <select id="month" v-model="monthSelected">
+                <option v-for="i in monthValues"
+                        :key=i
+                        :value=i>
+                  {{ i }}
+                </option>
+              </select>
+            </div>
+            <span class="ymd-text">月</span>
+
+            <div class="select" v-on:change="changeDate()">
+              <select id="day" v-model="daySelected">
+                <option v-for="i in dayValues"
+                        :key=i
+                        :value=i>
+                  {{ i }}
+                </option>
+              </select>
+            </div>
+            <span class="ymd-text">日</span>
+
+          </div>
+        </div>
+      </div>
+
+      <div class="container__footer">
+        <router-link class="button is-primary" to="/question">
+          次へ進む
+          <i class="fas fa-angle-right fa-fw"></i>
+        </router-link>
+      </div>
+
+    </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      // 本日の年月日を設定
+      year: new Date().getFullYear(),
+      month: new Date().getMonth() + 1,
+      day: new Date().getDate(),
+      // 年月日プルダウン要素の初期値設定
+      yearValues: [],
+      monthValues: 12,
+      dayValues: null,
+      // 年月日プルダウンの初期選択の値設定
+      yearSelected: null,
+      monthSelected: null,
+      daySelected: null,
+    };
+  },
+  mounted() {
+    // 年月日のセレクトボックス初期値設定
+    this.yearSelected = this.year;
+    this.monthSelected = this.month;
+    this.daySelected = this.day;
+    // 年日のセレクトボックスの要素設定
+    this.yearValues = this.genereateYear(this.year);
+    this.dayValues = this.genereateDay();
+  },
+  methods: {
+    // 西暦 (和暦)の形式で、現在から過去100年分の年の値を配列に格納
+    genereateYear(nowYear) {
+      const yearValues = [];
+      for (let i=nowYear-100; i<=nowYear; i++) {
```

Comment:
`=`と`<`は前後にスペースを空けるのが通例です。
エディターの自動整形を使用するといいですね。
```suggestion
      for (let i = nowYear-100; i <= nowYear; i++) {
```